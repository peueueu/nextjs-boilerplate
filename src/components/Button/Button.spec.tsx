import React from 'react'
import { screen, render } from '@testing-library/react'
import Button from './Button'

describe('<Button />', () => {
  it('should render the heading with the default values if no props are passed', () => {
    const { container } = render(<Button />)

    expect(screen.getByRole('heading', { name: /Button/i })).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
