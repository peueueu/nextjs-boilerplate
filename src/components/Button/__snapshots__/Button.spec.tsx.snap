// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> should render the heading with the default values if no props are passed 1`] = `
<main
  class=""
>
  <h1>
    Button
  </h1>
</main>
`;
